docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -143,6 +143,52 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
+If content is sourced from one location, and published in multiple locations, the `docs/reference` shortcode helps you control the appropriate link location.
+
+For example, alerting documentation is sourced from the Grafana repository and published to both Grafana documentation and Grafana Cloud documentation.
+Any link from alerting documentation to dashboard documentation, which is also published to both Grafana and Grafana Cloud, needs to be specific as to which destination is appropriate for the audience.
+
+Grafana Cloud alerting documentation should link to Grafana Cloud dashboard documentation, and Grafana alerting documentation should link to Grafana dashboard documentation.
```

Comment:
I think this sentence should be part of the previous paragraph.

Reply:
Fixed in https://github.com/grafana/writers-toolkit/pull/264/commits/9019ee670022e495abaa7cf055c3e3fa5e2df2d6